### PR TITLE
Closes #277: Column scrolling, if there are more notes then fit on the screen vertically

### LINKED ISSUE #277: If there are more notes then fit on the screen vertically, there should be a column scrolling, so you can see all notes

### PR DESCRIPTION
**Pull request by alphapfote**

### CODE REVIEW

File: src/components/Column/Column.scss [hunk 1]
```diff
@@ -58,10 +63,15 @@
...
+  overflow: auto;
+}
+
 .column__note-list {
-  padding: 0;
+  margin: 0;
+  padding: 0 $padding--large;
   display: grid;
-    grid-template-columns: repeat(auto-fill,minmax(260px,1fr));
+    grid-template-columns: repeat(auto-fill);
```
Comment (brandstetterm) [inline src/components/Column/Column.scss:74]: Regarding to my Chrome DevTools that's a `Invalid property value`

File: src/components/Column/Column.tsx [hunk 2]
```diff
@@ -29,11 +29,13 @@ const Column = ({ color, columnId } : ColumnProps) => {
...
                     <NoteInput columnId={columnId}/>
                 </header>
-                <ul className="column__note-list">
-                    {notesInColumn.map((note:any, index:number) =>
-                        <Note key={note.id} text={note.text} authorId={note.author}/>
-                    )}
-                </ul>
+                <div className="column__notes-wrapper">
+                    <ul className="column__note-list">
+                        {notesInColumn.map((note:any, index:number) =>
```
Comment (brandstetterm) [inline src/components/Column/Column.tsx:34]: Personally I really dislike the usage of `any`. Maybe you could use the `NoteClientModel` type here

File: src/components/Column/Column.scss (hunk 1 above)
Comment (alphapfote) [reply src/components/Column/Column.scss:74]: I reversed the changes, they have no effect

**Review by bitionaire — APPROVED** [review]

File: src/components/Column/Column.scss (hunk 1 above)
Comment (bitionaire) [reply src/components/Column/Column.scss:74]: Problem seems to be back

**Review by bitionaire — CHANGES_REQUESTED** [review]

**Review by bitionaire — APPROVED** [review]

**Pull request merged**